packages/@react-aria/tooltip/test/useTooltip.test.js

```diff
@@ -10,18 +10,211 @@
  * governing permissions and limitations under the License.
  */
 
+import {act, fireEvent, render} from '@testing-library/react';
 import React from 'react';
-import {renderHook} from '@testing-library/react-hooks';
-import {useTooltip} from '../';
-
-describe.skip('useTooltip', function () {
-  let renderTooltipHook = (props) => {
-    let {result} = renderHook(() => useTooltip(props));
-    return result.current;
-  };
-
-  it('handles defaults', function () {
-    let {tooltipProps} = renderTooltipHook({children: 'Test Tooltip'});
-    expect(tooltipProps.role).toBe('tooltip');
+import {useInteractionModality} from '@react-aria/interactions';
+import {useTooltip, useTooltipTrigger} from '../';
+import {useTooltipTriggerState} from '@react-stately/tooltip';
+
+describe('useTooltip', function () {
+  afterEach(() => {
+    jest.useRealTimers();
+  });
+  it('opens tooltip on hover after delay', function () {
+    jest.useFakeTimers('modern');
+
+    function Test() {
+      useInteractionModality();
```

Comment:
Having to do that (or using `useFocusVisible` which also leads to `setupGlobalFocusEvents`) is IMHO a bug - but it's a separate issue so I think it would be best to create a ticket for it and handle it later.

Reply:
Filed an issue here: https://github.com/adobe/react-spectrum/issues/1378